core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -205,6 +204,8 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal
     tws.map(_ => Nil)(akka.dispatch.ExecutionContexts.sameThreadExecutionContext)
   }
 
+  protected def generateUUID(pr: PersistentRepr): UUID = UUIDs.timeBased()
```

Comment:
We should perhaps have more clear extension points than extending the `CassandraJournal`, but this is ok for now.